api/room_resource/schema.py

```diff
@@ -4,6 +4,9 @@
 from graphql import GraphQLError
 
 from api.room_resource.models import Resource as ResourceModel
+from api.room.models import RoomResource as RoomResourceModel
+from api.room.models import Room as RoomModel
+from api.room.schema import Room as RoomSQLAlchemyObject
```

Comment:
Why is the room class being imported as `RoomSQLAlchemyObject`?

Reply:
This is mainly to avoid confusion as there exists other sqlalchemy objects in the file